src/status.rs

```diff
@@ -172,3 +167,12 @@ pub fn status(
         CryptStatusInfo
     )
 }
+
+/// Get size of encryption sectors in bytes
```

Comment:
The cryptsetup docs use "sector" singular. Should you be doing the same? I can't figure out what this value means from the crypt docs. I'ld like it to mean some useful information about the size of some crypt metadata. Maybe it describes the bytes taken up by all the LUKS2 metadata?

Reply:
Ok. This is really the size of the sectors on the device, hence the size of the chunks of encrypted data.